src/services/cronJob.ts

```diff
@@ -0,0 +1,18 @@
+import cron from "node-cron";
+import updateExpiredProducts from "../helpers/updateExpiredProducts";
+import { Request, Response } from "express";
+
+const cronSchedule = "* * * * *";
```

Comment:
why do you use environment variables so we can modify this variable `cronSchedule`